### PR TITLE
Hackathon: Add YouTube embeds, list working groups with slack channels

### PR DESCRIPTION
**Pull request by ewels**
localhost_8888_events_2020_hackathon-july-2020 (1)

### CODE REVIEW

File: markdown/events/2020/hackathon-july-2020.md [hunk 1]
```diff
@@ -57,20 +57,35 @@ To participate, all you need is a computer with an internet connection.
...
+
+* [`#hackathon-july-2020-new-pipelines`](https://nfcore.slack.com/channels/hackathon-july-2020-new-pipelines) - New pipeline development
+* [`#hackathon-july-2020-existing-pipelines`](https://nfcore.slack.com/channels/hackathon-july-2020-existing-pipelines) - Existing nf-core pipeline development
+* [`#hackathon-july-2020-tools`](https://nfcore.slack.com/channels/hackathon-july-2020-tools) - Improving nf-core framework tools (written in Python)
+* [`#hackathon-july-2020-modules-dsl2`](https://nfcore.slack.com/channels/hackathon-july-2020-modules-dsl2) - Nextflow DSL2 modules
+
+Tasks, in the form of GitHub issues, can be gathered and organised within the hackathon project board on GitHub:
 [https://github.com/orgs/nf-core/projects/6](https://github.com/orgs/nf-core/projects/6)
 
+If you'd like to form sub-groups, for example to work on a specific pipeline, that's fine.
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:86]:
```suggestion
For the wrap-up daily sessions, the progress made during the day will presented in the form of HackMD slides. To show your participation in the Hackathon groups, you can contribute to the slides. Each group has its own slides deck:

* New pipelines slides: [Edit](https://hackmd.io/U3hNik2AQG2bor_4pWlvfQ), [View](https://hackmd.io/@nf-core-hackathon-july-2020/ByaotZZkv)
* Existing pipelines slides: [Edit](https://hackmd.io/sVE8c7XxSW-QyCBaDAwxMg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/Hk7HqW-JD)
* Tools slides: [Edit](https://hackmd.io/Okx7xOUvTGW4NMVropSttg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/r1ZJiZWkP)
* DSL2 modules slides: [Edit](https://hackmd.io/CMmZaDl2QWK3C7ZBDzffPw), [View](https://hackmd.io/@nf-core-hackathon-july-2020/By4n2-bkD)

If you'd like to form sub-groups, for example to work on a specific pipeline, that's fine.
```

Comment (ggabernet) [reply markdown/events/2020/hackathon-july-2020.md:86]: These are the links to the HackMD slides. Core members can also see the [team board](https://hackmd.io/team/nf-core-hackathon-july-2020?nav=overview) and create new notes, and logged in participants can edit and view the slides links.

File: markdown/events/2020/hackathon-july-2020.md [hunk 2]
```diff
@@ -140,6 +155,8 @@ If you're in a different time zone, [this time-zone converter tool](https://savv
 
 ### Hackathon Introduction
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/nzk4qzSFBb4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:159]:
```suggestion
Join the [Zoom Meeting](https://zoom.us/j/99218495584) for this session.

```

File: markdown/events/2020/hackathon-july-2020.md [hunk 3]
```diff
@@ -182,13 +199,17 @@ _Tuesday 14th, 14:00 - 14:30_
 
 ### Data management solutions
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/u1kg8HTimXk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:203]:
```suggestion

Join the [Zoom Meeting](https://zoom.us/j/96839947521) for this session.

```

File: markdown/events/2020/hackathon-july-2020.md [hunk 4]
```diff
@@ -182,13 +199,17 @@ _Tuesday 14th, 14:00 - 14:30_
...
 Elin Kronander ([@elinkronander](https://github.com/elinkronander))<br>
 _Wednesday 15th, 13:00 - 13:30_
 
 ---
 
 ### nf-core tools & guidelines
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Xl_bCDeqaUY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:212]:
```suggestion

Join the [Zoom Meeting](https://zoom.us/j/96822836533) for this session.

```

File: markdown/events/2020/hackathon-july-2020.md [hunk 5]
```diff
@@ -217,5 +238,7 @@ _Friday 17th, 13:40 - 14:00_
 
 ### Final wrap-up and closing remarks
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/e-eEWFM0ozs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:242]:
```suggestion

Join the [Zoom Meeting](https://zoom.us/j/97710057433) for this session.

```

File: markdown/events/2020/hackathon-july-2020.md [hunk 6]
```diff
@@ -57,20 +57,42 @@ To participate, all you need is a computer with an internet connection.
...
+Tasks, in the form of GitHub issues, can be gathered and organised within the hackathon project board on GitHub:
 [https://github.com/orgs/nf-core/projects/6](https://github.com/orgs/nf-core/projects/6)
 
+For the wrap-up daily sessions, the progress made during the day will presented in the form of HackMD slides. To show your participation in the Hackathon groups, you can contribute to the slides. Each group has its own slides deck:
+
+* New pipelines slides: [Edit](https://hackmd.io/U3hNik2AQG2bor_4pWlvfQ), [View](https://hackmd.io/@nf-core-hackathon-july-2020/ByaotZZkv)
+* Existing pipelines slides: [Edit](https://hackmd.io/sVE8c7XxSW-QyCBaDAwxMg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/Hk7HqW-JD)
+* Tools slides: [Edit](https://hackmd.io/Okx7xOUvTGW4NMVropSttg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/r1ZJiZWkP)
+* DSL2 modules slides: [Edit](https://hackmd.io/CMmZaDl2QWK3C7ZBDzffPw), [View](https://hackmd.io/@nf-core-hackathon-july-2020/By4n2-bkD)
```
Comment (ggabernet) [inline markdown/events/2020/hackathon-july-2020.md:91]:
```suggestion
```

**Comment by ewels** [comment]
Nice! Changes applied and then tweaked a little (I can't resist a good icon). Now looks like this:

image

**Comment by ewels** [comment]
Easy to add them when they're ready :)

**Pull request merged**